Automattic/jetpack
# Launchpad: Add "Customize welcome message" task for Newsletter post-launch task lists

Fixes https://github.com/Automattic/wp-calypso/issues/79563

## Proposed changes:

* Add a "Customize welcome message" task to both Newsletter paid and free post-launch task lists.
* Add a logic function hooked to `update_option_subscription_options` to see if the `invitation` subscription option has changed, and if so, mark the task as complete.
* Combine with this front-end PR to see the task in action: https://github.com/Automattic/wp-calypso/pull/79792

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:






* Apply this patch to your sandbox and sandbox public-api
* Visit the developer console and set it to WP REST API `wpcom/v2/sites/siteSlug/launchpad?checklist_slug=intent-paid-newsletter` and `wpcom/v2/sites/siteSlug/launchpad?checklist_slug=intent-free-newsletter`
* You should see the `customize_welcome_message` task in the list of tasks in the response.

<img width="1045" alt="Screenshot 2023-07-24 at 10 26 48 AM" src="https://github.com/Automattic/jetpack/assets/2124984/01d68e92-096c-4d7c-9353-b6f34f002c22">
